test/project_types/script/layers/infrastructure/languages/assemblyscript_project_creator_test.rb

```diff
@@ -9,7 +9,7 @@
   let(:language) { "AssemblyScript" }
   let(:script_id) { "id" }
   let(:context) { TestHelpers::FakeContext.new }
-  let(:extension_point_type) { "discount" }
+  let(:extension_point_type) { "payment_methods" }
```

Comment:
discount is set to `deprecated: true` so I swapped this to a more recent extension point.